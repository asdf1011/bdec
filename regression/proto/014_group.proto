message Groups {
  repeated group Hello = 1 {
    required int32 a = 1;
  };
  optional group World = 2 {
    required string b = 1;
    required string c = 2;
  }
}
