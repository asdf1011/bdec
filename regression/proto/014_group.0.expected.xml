<Groups>
    <Hello>
        <Hello>
            <a>5</a>
        </Hello>
        <Hello>
            <a>7</a>
        </Hello>
        <Hello>
            <a>9</a>
        </Hello>
    </Hello>
    <World>
        <b>chicken</b>
        <c>little</c>
    </World>
</Groups>
